Documentation/ceph-cluster-crd.md

```diff
@@ -84,8 +84,12 @@ This feature is only available when `useAllNodes` has been set to `false`.
 
 ### Mon Settings
 
-- `count`: set the number of mons to be started. The number should be odd and between `1` and `9`. If not specified the default is set to `3` and `allowMultiplePerNode` is also set to `true`.
-- `allowMultiplePerNode`: enable (`true`) or disable (`false`) the placement of multiple mons on one node. Default is `false`.
+- `count`: Set the number of mons to be started. The number should be odd and between `1` and `9`. If not specified the default is set to `3` and `allowMultiplePerNode` is also set to `true`.
+- `preferredCount`: If you want to increase the number of mons when the number of nodes increases, set the `preferredCount` to be larger than the `count`. For example, if your cluster
+starts with three nodes, but might grow to more than five nodes, you might want five mons running after the other nodes come online. In this case, set `count: 3` and `preferredCount: 5`.
```

Comment:
The same maximum of `9` applies to `prefferredCount` as is currently with `count` already, in the current operator side CRD validation, right?

Reply:
Yes, the validation is found in the crd definition for a max of 9. The minimum is defined as 0 meaning that it's not set and we will use the `count` instead